gnocchi/rest/api.py

```diff
@@ -1890,6 +1899,118 @@ class BatchController(object):
     resources = ResourcesBatchController()
 
 
+class PrometheusWriteController(rest.RestController):
```

Comment:
Not a big issue but is there any reason you did not put that into `gnocchi.rest.prometheus` to not make that file too big?